patch_parser_test.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from patch_parser import parse_comments
+from archive_converter import generate_email_from_file
+
+class Patch_Parser_Test(unittest.TestCase):
+
+    def test_parse_comments_for_single_email_thread(self):
```

Comment:
Can you also add test cases for:

1. Multiple patches, no cover letter.
2. Multiple patches with cover letter.
3. Email no patches.

Reply:
We talked offline and I agree that the other test cases I asked for will not work at this point since there is no "index" to build up email threads.

Since adding the index is outside the scope of this change, I am fine with just putting a TODO comment here as a reminder to add the other test cases when possible.